src/main/java/Duke.java

```diff
@@ -1,10 +1,56 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    static ArrayList<Task> list = new ArrayList<Task>();
+
+    public static void printTasks() {
+        System.out.println("Here are the tasks in your list:");
+        for (Task task : list) {
+            System.out.println(String.format("%d. %s", task.getTaskId(), task.toString()));
+        }
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
+        String logo = " ____         _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+        Scanner scan = new Scanner(System.in);
+        String userInput = scan.nextLine();
+        while (!userInput.equals("bye")) {
+            if (userInput.equals("list")) {
+                printTasks();
+            } else if (userInput.length() > 4 && userInput.substring(0, 5).equals("done ")) {
```

Comment:
Consider simplifying the conditional statement using startsWith. It is applicable to the rest of the conditions below.
Ex. else if(userInput.startsWith("done")

Reply:
Thank you for your suggestion. I find it very efficient 👍 Will add it to my code later on.